mon-pix/app/styles/components/_circle-chart.scss

```diff
@@ -14,9 +14,16 @@
   height: 100px;
   max-height: 128px;
 
+
   &--big {
-    width: 120px;
-    height: 120px;
+    width: 70px;
+    height: 70px;
+  }
+  @include device-is('tablet') {
```

Comment:
pourquoi ne pas mettre `@include device-is('tablet')` dans le premier `&--big` ?